hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -513,6 +513,7 @@
         ram: {
           label:    "ram_ret_aon",
           swaccess:   "rw",
+          data_intg_passthru: "true",
```

Comment:
as a separate note, if we add this comment here, we should probably also long term consider how we can pass this directly into the related modules to manipulate the options on the `tlul_adapter_sram`.  Otherwise things could get out of sync.

These tests would catch that though, so probably not a super high priority.

Reply:
Agree. This is so far used in DV RAL model. Design should pass this to the tlul_adapter_sram from the top.